src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.props

```diff
@@ -14,6 +14,16 @@ Copyright (c) .NET Foundation. All rights reserved.
     Properties and tasks supporting Razor MSBuild integration
   -->
 
+  <!--
+    Properties for static assets in the Razor SDK.
+  -->
+  <PropertyGroup>
+    <DefaultItemExcludes>$(DefaultItemExcludes);**\node_modules\**;node_modules\**</DefaultItemExcludes>
```

Comment:
Is there any risk of this causing build/VS perf issues? It looks like this will make MSBuild enumerate every single file under `node_modules` if there is one, storing a big list in memory, and that directory could easily contain over 100,000 files.

Not that I know what a better alternative is.

Reply:
Oh no wait, that doesn't happen here since this is a property, not an itemgroup

Reply:
Yeah, also this comes from the web sdk.